data/json/items/gun/50.json

```diff
@@ -70,5 +70,29 @@
       "barrel_length": -1
     },
     "flags": [ "RELOAD_EJECT" ]
+  },
+  {
+    "id": "spr_2",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "Pindad SPR 2",
+    "name_plural": "Pindad SPR-2"
```

Comment:
Missing a comma here.

Reply:
im sorry for the mistakes, im still learning the JSON codes based on available material on the list and try to create one based on the material, so should i fix it now?